src/components/AuthButton/AuthButton.jsx

```diff
@@ -1,17 +1,33 @@
 import React from 'react';
 import {Button} from 'react-bootstrap';
+import DropdownButton from 'react-bootstrap/DropdownButton';
+import Dropdown from 'react-bootstrap/Dropdown';
 import {Link} from "react-router-dom";
 import {logout} from '../../store/actions/login'
 import { useDispatch, useSelector } from 'react-redux';
 
 const AuthButton = () => {
     const dispatch = useDispatch()
     const logged = useSelector(state => state.loginReducer.logged)
+    const user = useSelector(state => state.loginReducer.user)
 
     const handleLogout = () => dispatch(logout());
 
+    const formatTitle = () => {
+      return <a>
```

Comment:
the a tag was throwing a warning because it had no href.
added an href + prevented default behavior on click. not the best solution but we cant change it to a button inside the title